src/buildercore/fastly/vcl/journal-google-scholar-vary.vcl

```diff
@@ -0,0 +1,6 @@
+if (!req.http.Fastly-FF && !req.http.Fastly-Debug) {
+    set resp.http.Vary = regsub(resp.http.Vary, "X-eLife-Google-Scholar-Metadata, ", "");
+    if (resp.http.Vary ~ "^\s*$") {
+        unset resp.http.Vary;
+    }
+}
```

Comment:
not sure what does this do, shouldn't be `Vary` be purely managed by the origin?

Reply:
Stops it being sent to the client.

Reply:
Because it's related to an header that the client didn't send, that makes sense. This is still end-to-end testable by setting `User-Agent` manually and going through the CDN.